lab_1/main.py

```diff
@@ -116,35 +110,40 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
         for i in top_n_words_unknown_profile:
             if i in top_n_words_known_profile:
                 top_n_common_words.append(i)
-        intersecting_words = len(top_n_common_words) / len(top_n_words_unknown_profile)
-        return round(float(intersecting_words), 2)
+        distance = len(top_n_common_words) / len(top_n_words_unknown_profile)
+        return round(float(distance), 2)
     return None
```

Comment:
None check to the beginning

Reply:
oh, ok,i didn't noticed that. now it will be improved. thanks for cheking! and i understand what the loops mean!